godtools/App/Services/Renderer/Views/MobileContent/Views/ContentText/MobileContentTextViewModel.swift

```diff
@@ -58,7 +49,7 @@ class MobileContentTextViewModel: MobileContentTextViewModelType {
     }
     
     var hidesStartImage: Bool {
-        guard let resource = textModel.startImage else {
+        guard let resource = startImageResourceName else {
             return true
         }
         return resource.isEmpty
```

Comment:
this could be simplified to just be based off of `textModel.startImage == null`

Reply:
similar can be done for the end image (if that logic exists)

Reply:
@frett  I like that better.